kong/db/strategies/cassandra/connector.lua

```diff
@@ -301,4 +301,28 @@ function CassandraConnector:truncate()
 end
 
 
+function CassandraConnector:truncate_table(table_name)
+  local ok, err = self:connect()
```

Comment:
I believe the `:connect()` and `:setkeepalive()` calls aren't necessary, since `:query()` will do these for you? If there are no loops anyway, I think we should stay away from this pattern and let `:query()` do the work.